### ISSUE TITLE
Documentation amendment about bumpversion

### ISSUE THREAD

**Issue by manikos** [twoscoops/Creating-and-Distributing-Python-Packages#11]
## Location within the Course

* Section: 4
* Lesson: 5
* Language: en

## Description 
Because `bumpversion` does a lot of magic under the hood, in the `Auto-Increment the Version Number` section, better change this line:

    bumpversion minor

into this:

    bumpversion --no-tag --no-commit minor

`--no-tag`: Do not create a tag in version control
`--no-commit`: Do not commit to version control

Because, if I change my code and I need to update the version and do `bumpversion minor`, then the `git status` says:

    On branch master
    nothing to commit, working directory clean

Thus the `--no-commit` argument above.

Also, the `git tag` says:

    0.1.0
    v0.2.0

Which is not quite pretty. We don't need the `v` prefix, which `bumpversion` add automatically. Thus, the `--no-tag` argument above.

**Comment by purplediane** [comment]
Yes, I was very confused at first, because my file changed but git said everything was fine. I thought that somehow the file changed and git didn't know it! Was worried there for a while until I figured out that bumpversion committed the change and all I needed to do was push the changes. :-)

We definitely need more explanation of what is going on here, even if we don't change the actual commands used.
